hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -298,4 +318,18 @@ private void dispatchAndMarkCreation(
 
         tokenServiceApi.markAsContract(accountId, autoRenewAccountId);
     }
+
+    public void externalizeHollowAccountMerge(@NonNull ContractID contractId, @Nullable Bytes evmAddress) {
+        var recordBuilder = context.addRemovableChildRecordBuilder(ContractCreateRecordBuilder.class);
+        recordBuilder
+                .contractID(contractId)
+                // add dummy transaction, because SingleTransactionRecord require NonNull on build
+                .transaction(Transaction.newBuilder()
```

Comment:
I think this will need to be a synthetic `ContractUpdate`; but that will be much easier to test once [this PR](https://github.com/hashgraph/hedera-services/pull/9513) is merged.